framework/resources/Functional/functions/functions.json

```diff
@@ -0,0 +1,22 @@
+{
+  "testId": "functions",
```

Comment:
Hey @agozhiy I would suggest retaining the new "functions" dir and creating a sub dir for coalesce function. The tests here can be moved inside that. And correspondingly the testID here can be updated to be more unique to it.

Reply:
Done.